### PR TITLE
Remove process bellatrix payload

### PR DESCRIPTION
**Pull request by avalonche**
## 📚 References

---

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test-race`
* [x] `go mod tidy`

### CODE REVIEW

File: server/service.go [hunk 1]
```diff
@@ -221,22 +221,6 @@ func (m *BoostService) sendValidatorRegistrationsToRelayMonitors(payload []types
 	}
 }
 
-func (m *BoostService) sendAuctionTranscriptToRelayMonitors(transcript *AuctionTranscript) {
```
Comment (metachris) [inline server/service.go:224]: why remove this?

File: server/service.go [hunk 2]
```diff
@@ -655,9 +525,6 @@ func (m *BoostService) processCapellaPayload(w http.ResponseWriter, req *http.Re
 		log.Warn("bid found but no associated relays")
 	}
 
-	// send bid and signed block to relay monitor with capella payload
-	// go m.sendAuctionTranscriptToRelayMonitors(&AuctionTranscript{Bid: originalBid.response.Data, Acceptance: payload})
```
Comment (metachris) [inline server/service.go]: better keep around

**Comment by metachris** [comment]
looks great on quick check! but let's keep the relay monitor stuff around for now

**Comment by avalonche** [comment]
as the relay monitor code is not used, it's failing linting. instead of removing I'll just comment out the relay monitor related code for now.

**Review by metachris — APPROVED** [review]

**Pull request merged**